src/applications/check-in/pre-check-in/tests/e2e/happy-path/happy.path.cypress.spec.js

```diff
@@ -46,10 +46,10 @@ describe('Pre-Check In Experience ', () => {
     // page: Next of Kin
     cy.get('h1', { timeout: Timeouts.slow })
       .should('be.visible')
-      .and('have.text', 'Next of Kin');
+      .and('have.text', 'Is this your current next of kin?');
```

Comment:
You should use the next of kin helper page class here.

Reply:
ah yeah not sure why that went over my head as I was updating that!